tests/deployments.spec.ts

```diff
@@ -1,14 +1,14 @@
 import { beforeEach, expect, test } from "@jest/globals";
 import { constructPaginationString } from "../src/utils/pagination";
-import { getDeploymentsList } from "../src/deployments";
+import { listDeployment } from "../src/deployments";
 import { setVercelToken } from "../src";
 
 beforeEach(() => {
   setVercelToken(process.env?.VERCEL_TOKEN as string);
 });
 
 test("calling just the get deployment endpoint", async () => {
-  const { response, error } = await getDeploymentsList();
+  const { response, error } = await listDeployment();
```

Comment:
Once you change the function name in the source, you'll have to change it here as well to `listDeployments`